wandb/sdk/launch/builder/build.py

```diff
@@ -248,6 +248,8 @@ def get_env_vars_dict(launch_project: LaunchProject, api: Api) -> Dict[str, str]
         env_vars["WANDB_USERNAME"] = launch_project.launch_spec["author"]
     if launch_project.sweep_id:
         env_vars["WANDB_SWEEP_ID"] = launch_project.sweep_id
+    if launch_project.launch_spec.get("resume"):
```

Comment:
Shouldn't this be if `_resume_count` is greater than 0?

Reply:
Thanks, this got messed up with a merge conflict.